README.md

```diff
@@ -27,8 +21,7 @@
 
 #### Acknowledgements
 
-* Some parts of this sample application were inspired by the excellent 
-  [Java FX tutorial](http://code.makery.ch/library/javafx-8-tutorial/) by *Marco Jakob*. 
+The sample project created by SE-EDU initiative at https://github.com/se-edu/
```

Comment:
hmmm.. you need to ack addressbook-level4 in this, since that is where you fork from and spin off your project.